handler.js

```diff
@@ -400,7 +400,7 @@ class GithubBuild {
       tests.push({
         name: "backend",
         type: "uat",
-        deployable: true
+        deployable: false
```

Comment:
Should this still `true` or add a new deployment task?

Reply:
We've added now a seperate task for deployment only. We thought this makes things easier.

Reply:
I will do the refactoring @tuanmh.